README.md

```diff
@@ -73,17 +73,45 @@ UI5 linter report:
 Note: Use "ui5lint --details" to show more information about the findings
 ```
 
-### Options
+You can provide multiple glob patterns as arguments after the `ui5lint` command to filter and narrow down the linting results.
 
-#### `--file-paths`
+```sh
+> ui5lint webapp/**/*
```

Comment:
```suggestion
> ui5lint 'webapp/**/*'
```

Using `webapp/**/*` will expand the glob in the shell (before ui5lint is executed). The linter will receive a list of all files matching the pattern as an argument

Using `'webapp/**/*` will pass the pattern as an argument to ui5lint, which then passes it to _our_ glob function.

I'm not entirely sure which way is better, but I think the latter is more likely to behave the same across platforms and shells, so maybe we should document that. @codeworrior do you have an opinion on this?

Reply:
Maybe it makes sense to show an example where less files than usual are linted? Something like `webapp/**/*.xml` to only check XMLs maybe? Then report example below needs to be updated as well in that case.

Reply:
Docs are updated :)